### PR TITLE
Add addNewItem button

### PR DESCRIPTION
**Pull request by JanekJaniak**

### CODE REVIEW

File: src/components/RenderList.vue [hunk 1]
```diff
@@ -4,27 +4,27 @@
...
+            <label for = "price">Price/Kg:</label>
+            <input id = "price" placeholder="Edit" v-model.number="ingredient.price" type ="number" />
+            <label>VAT:</label>
+            <select v-model="ingredient.vat" >
+              <option v-for="option in vatOptions" :value="option.value" :key="option.id" :disabled="option.disabled" >
+                {{option.text}}
+              </option>
+            </select>
+            <strong>Net Value:{{calculateNetValue(ingredient.quantity,ingredient.price)}}</strong>
+            <strong>Gross Value:{{calculateGrossValue(ingredient.quantity,ingredient.price,ingredient.vat)}}</strong>
```
Comment (jan-gajdowicz) [inline src/components/RenderList.vue:19]: use spaces between arguments

File: src/components/RenderList.vue [hunk 2]
```diff
@@ -76,11 +76,18 @@ export default {
       },
       handleInput(event) {
         this.value = event.target.value;
-        console.log(this.value);
```
Comment (jan-gajdowicz) [inline src/components/RenderList.vue:79]: Remove unused method

**Review by jan-gajdowicz — CHANGES_REQUESTED** [review]
2 minor things

File: src/components/RenderList.vue (hunk 1 above)
Comment (JanekJaniak) [reply src/components/RenderList.vue:19]: ✔️

File: src/components/RenderList.vue (hunk 2 above)
Comment (JanekJaniak) [reply src/components/RenderList.vue:79]: ✔️

**Pull request merged**